content/en/docs/concepts/cluster-administration/node-autoscaling.md

```diff
@@ -0,0 +1,289 @@
+
+The specifics of the provisioned Nodes (for example the amount of resources, the presence of a given
+label) depend on autoscaler configuration. Autoscalers can either choose them from a pre-defined set
+of Node configurations, or use [auto-provisioning](#autoprovisioning).
+
+### Auto-provisioning {#autoprovisioning}
+
+Node auto-provisioning is a mode of provisioning in which a user doesn't have to fully configure the
+specifics of the Nodes that can be provisioned. Instead, the autoscaler dynamically chooses the Node
+configuration based on the pending Pods it's reacting to, as well as pre-configured constraints (for
+example, the minimum amount of resources or the need for a given label).
+
+## Node consolidation {#consolidation}
+
+The main component of the cost of running a cluster is usually the cost of the infrastructure (most
+commonly Virtual Machines) backing its Nodes. In order to minimize this cost and still be able to
+run all Pods in the cluster, the Pods should utilize as much of the Nodes' resources as possible.
+From this perspective, the overall Node utilization in a cluster can be used as a proxy for how
+cost-effective the cluster is.
+
+{{< note >}}
+Correctly setting the resource requests of your Pods is as important to the overall
+cost-effectiveness of a cluster as optimizing Node utilization.
+Combining Node autoscaling with [vertical workload autoscaling](#vertical-workload-autoscaling) can
+help you achieve this.
+{{< /note >}}
+
+Nodes in your cluster can be automatically _consolidated_ in order to improve the overall Node
+utilization, and in turn the cost-effectiveness of the cluster. Consolidation happens through
+removing a set of underutilized Nodes from the cluster. Optionally, a different set of Nodes can
+be [provisioned](#provisioning) to replace them.
+
+Consolidation, like provisioning, only considers Pod resource requests and not real resource usage
+when making decisions.
+
+For the purpose of consolidation, a Node is considered _empty_ if it only has DaemonSet and static
+Pods running on it. Removing empty Nodes during consolidation is more straightforward than non-empty
+ones, and autoscalers often have optimizations designed specifically for consolidating empty Nodes.
+
+Removing non-empty Nodes during consolidation is disruptive&mdash;the Pods running on them are
+terminated, and possibly have to be recreated (for example by a Deployment). However, all such
+recreated Pods should be able to schedule on existing Nodes in the cluster, or the replacement Nodes
+provisioned as part of consolidation. __No Pods should normally become pending as a result of
+consolidation.__
+
+{{< note >}}
+Autoscalers predict how a recreated Pod will likely be scheduled after a Node is provisioned or
+consolidated, but they don't control the actual scheduling. Because of this, some Pods might
+technically become pending as a result of consolidation - if for example a completely new Pod
```

Comment:
```suggestion
become pending as a result of consolidation - if for example a completely new Pod
```

Would drop technically - non-zero chance of this actually happening.

Reply:
One might argue that in this scenario the pods became pending as a result of new pods appearing, not the initial consolidation - hence the technically. It seems that it's confusing without this further clarification though, dropping it sgtm